web-components/src/components/empty-state/EmptyState.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
```

Comment:
Thought I made a ticket to update these, but can’t find - appreciate you updating these (and using proper `.bind` syntax)